ResidentVulnerabilitiesDataPipeline/Database/DatabaseActions.cs

```diff
@@ -15,7 +15,7 @@ public int CopyDataToDatabase(ILambdaContext context, string awsRegion, string b
         {
             var loadDataCommand = _npgsqlConnection.CreateCommand();
 
-            var loadDataFromCSV = @"SELECT aws_s3.table_import_from_s3('qlik_vulnerability','','(FORMAT csv, HEADER)',@bucket, @objectkey, @awsregion);";
+            var loadDataFromCSV = @"SET datestyle = 'DMY'; SELECT aws_s3.table_import_from_s3('qlik_vulnerability','','(FORMAT csv, HEADER)',@bucket, @objectkey, @awsregion);";
```

Comment:
What's the reasoning for using this format over the ISO8601 format? i.e. YYYY-MM-DD

Reply:
The CSV file used has English formatted dates